chain/jsonrpc/client/src/lib.rs

```diff
@@ -234,6 +237,14 @@ impl JsonRpcClient {
         call_method(&self.client, &self.server_addr, "EXPERIMENTAL_changes", request)
     }
 
+    #[allow(non_snake_case)]
+    pub fn EXPERIMENTAL_congestion_level(
+        &self,
+        request: RpcCongestionLevelRequest,
+    ) -> RpcRequest<RpcCongestionLevelResponse> {
+        call_method(&self.client, &self.server_addr, "EXPERIMENTAL_congestion_level", request)
+    }
+
```

Comment:
Does this mean that the congestion level isn't currently available via json rpc?